uai2017/python/superuser-init.py

```diff
@@ -0,0 +1,338 @@
+            readers=['UAI.org/2017/pcs','UAI.org/2017'],
+            writers=['UAI.org/2017/pcs'],
+            signatures=['UAI.org/2017/pcs'],
+            signatories=['UAI.org/2017/conference/reviewers-invited'],
+            members=[])
+        groups.append(uai2017reviewersinvited)
+
+    if overwrite_allowed('UAI.org/2017/conference/reviewers-emailed'):
+        uai2017reviewersemailed = Group('UAI.org/2017/conference/reviewers-emailed',
+            readers=['UAI.org/2017/pcs','UAI.org/2017'],
+            writers=['UAI.org/2017/pcs'],
+            signatures=['UAI.org/2017/pcs'],
+            signatories=['UAI.org/2017/conference/reviewers-emailed'],
+            members=[])
+        groups.append(uai2017reviewersemailed)
+
+
+    if overwrite_allowed('UAI.org/2017/conference/reviewers'):
+        uai2017reviewers = Group('UAI.org/2017/conference/reviewers',
+            readers=['everyone'],
+            writers=['UAI.org/2017/conference','UAI.org/2017/pcs'],
+            signatures=['UAI.org/2017/conference'],
+            signatories=['UAI.org/2017/conference/reviewers'],
+            members=[])
+        groups.append(uai2017reviewers)
+
+
+    if overwrite_allowed('UAI.org/2017/conference/reviewers-declined'):
+        uai2017reviewersdeclined = Group('UAI.org/2017/conference/reviewers-declined',
+            readers=['everyone'],
+            writers=['UAI.org/2017/conference','UAI.org/2017/pcs'],
+            signatures=['UAI.org/2017/conference'],
+            signatories=['UAI.org/2017/conference/reviewers'],
+            members=[])
+        groups.append(uai2017reviewersdeclined)
+
+
+    ## Post the groups
+    for g in groups:
+        print "Posting group: ",g.id
+        openreview.post_group(g)
+
+    openreview.post_group(openreview.get_group('host').add_member('UAI.org/2017'))
+
+
+
+
+    #########################
+    ##  SETUP INVITATIONS  ##
+    ######################### 
+
+
+    ## Create the submission invitation
+    reply = {
+        'forum': None,
+        'replyto': None,
+        'readers': {
+            'description': 'The users who will be allowed to read the above content.',
+            'values': ['everyone']
+        },
+        'signatures': {
+            'description': 'How your identity will be displayed with the above content.',
+            'values-regex': '~.*'
+        },
+        'writers': {
+            'values-regex': '~.*'
+        },
+        'content': {
+            'title': {
+                'description': 'Title of paper.',
+                'order': 1,
+                'value-regex': '.{1,100}',
+                'required':True
+            },
+            'authors': {
+                'description': 'Comma separated list of author names, as they appear in the paper.',
+                'order': 2,
+                'value-regex': '[^,\\n]+(,[^,\\n]+)*',
+                'required':True
+            },
+            'author_emails': {
```

Comment:
Yes, I agree. The changes that need to be made are:

1. authors_emails --> authorids
2. change authors, authorids, and conflicts from "value-regex" to "values-regex". This change will cause these fields to be saved as arrays, instead of strings ( e.g "Michael Spector, Melisa Bok, Ankur Agarwal" --> ["Michael Spector","Melisa Bok","Ankur Agarwal"] )
3. the regex strings for the above fields should be changed so that they refer to each item in the array, instead of regex for the whole string. For now I think it's fine to just set these as /.*/ (match any character).